newIDE/app/src/InstancesEditor/HighlightedInstance.js

```diff
@@ -9,6 +9,15 @@ export default class InstancesSelection {
     this.highlightedInstance = null;
     this.highlightRectangle = new PIXI.Graphics();
     this.highlightRectangle.hitArea = new PIXI.Rectangle(0, 0, 0, 0);
+
+    this.tooltipBG = new PIXI.Graphics();
```

Comment:
Nitpicking: this should be named `this.tooltipBg` if stricly following camelCase, but I think it's better to be explicit and name it `this.tooltipBackground`.